pallets/liquidity-pools/src/message.rs

```diff
@@ -50,13 +89,13 @@ where
 	///
 	/// Directionality: Centrifuge -> EVM Domain.
 	AddPool {
-		pool_id: PoolId,
+		pool_id: u64,
```

Comment:
Why the change to raw types. I do not really like that.

Reply:
We could avoid that, but the message is who dictates the serialization format. If tomorrow we change the `PoolId` type in the runtime, then the message format will change silently. So this module should know exactly how to serialize/deserialize each type, because this module is the module that implements the spec.

Reply:
Yes, sounds good!